Base/Expression.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace FYP_ETL.Base
+{
+    class Expression
+    {
+        public DataTable expressionDt = new DataTable();
+        public DataTable mapDt = new DataTable();
```

Comment:
Let's leave the datatables uninitialized here, and initialize them in the constructor on line 74 before adding the columns